### DISCUSSION TITLE
v2/zh/config/frontmatter/layout

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#275] [Announcements]
# v2/zh/config/frontmatter/layout

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/config/frontmatter/layout.html

**Comment by lipengjun92** [comment]
sidebar: false
这个配置好像不生效

**Comment by timpcfan** [comment]
headerDepth > 2 时似乎没有效果，仍然渲染2级的标题
